docs/templates/static/intro.md.j2

```diff
@@ -10,6 +10,10 @@ Cisco Firepower Threat Defense (FTD) devices. Currently, three Ansible modules a
 FTD modules allow executing any API operations in form of Ansible plays. The modules configure virtual and 
 physical devices by sending HTTPS calls formatted according to the REST API specification.
 
+__NOTE__: _Cisco makes no guarantee that the API version included on this Firepower Threat Device (the “API”) will be compatible with future releases. Cisco, at any time in its sole discretion, may modify, enhance or otherwise improve the API based on user feedback._
```

Comment:
is there any sense to mode this disclaimer to file which can be just included here?

Reply:
Done.